RUM-12403 Adds documentation for iOS manual keeping/dropping traces. (#34163)
DataDog/documentation: content/en/tracing/trace_collection/dd_libraries/ios.md

{{% /tab %}}
{{< /tabs >}}

8. (Optional) Override a sampling decision by forcing a trace to be kept or dropped. For example, always keep a trace of a critical transaction, or always drop unnecessary, repetitive traces.

Manually keep a trace:

{{< tabs >}}
{{% tab "Swift" %}}
```swift
span.keepTrace()
```
{{% /tab %}}
{{% tab "Objective-C" %}}
[span keepTrace];
{{% /tab %}}
{{< /tabs >}}

Manually drop a trace:

{{< tabs >}}
{{% tab "Swift" %}}
```swift
span.dropTrace()
```
{{% /tab %}}
{{% tab "Objective-C" %}}
[span dropTrace];
{{% /tab %}}
{{< /tabs >}}

<div class="alert alert-danger">
  Always call `keepTrace()` or `dropTrace()` on a root span before child spans are created, and before any context propagation. Otherwise, the system can't ensure consistency, resulting in partial traces being ingested.
</div>

> 9. (Optional) To distribute traces between your environments, for example frontend - backend, you can either do it manually or leverage our auto instrumentation. In both cases, you can opt to inject the trace context into all requests or only into the sampled ones. A sampling of 100% is applied by default.

Reviewer: ```suggestion
9. (Optional) To distribute traces between your environments, for example from frontend to backend, you can either do it manually or use auto instrumentation. In both cases, you can opt to inject the trace context into all requests or only into the sampled ones. A sampling of 100% is applied by default.
```
